### ISSUE TITLE
invalid flag: --checksum on ADD

### ISSUE THREAD

**Issue by danilobuerger** [hadolint/hadolint#985]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

When using 
```
# syntax=docker/dockerfile:1
ADD --checksum=
```
I am getting `invalid flag: --checksum`.

### Actual behavior

hadolint should be able to parse this flag.

### Steps to reproduce the behavior

See above,.

**Output of `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 2.12.1
```

**Comment by ppettina** [comment]
This should fix it: https://github.com/hadolint/language-docker/pull/92

**Comment by kevin-kortum-trustedshops** [comment]
So a release would be nice? Otherwise it is still broken for me running 2.12.0

**Comment by thiagowfx** [comment]
A new release with the aforementioned fix would be much appreciated! :)
